data/successors/successor events.txt

```diff
@@ -86,3 +86,13 @@ event "successors: trust 2"
 		remove "Veusade (Naive)"
 		add "Aaulqra"
 		add "Veusade"
+
+event "successors: unified defense"
+	planet "Kasii-Tuur-Saqra"
+		government "New Houses"
```

Comment:
It might be easier to just change the reputation like you do with the other houses.